src/components/Layout/Header/Header.test.js

```diff
@@ -1,11 +1,36 @@
-import Header from './Header'
 import { render } from '@testing-library/react'
+import Header from './Header'
+import UserContext from '../../UserContext/UserContext'
 
 describe('Header', () => {
-  it('renders a Header with serviceName props', () => {
-    const serviceName = 'This is the Header'
-    const { getByText } = render(<Header serviceName={serviceName} />)
-    const header = getByText(serviceName)
-    expect(header).toBeInTheDocument()
+  const serviceName = 'Hackney Header'
+
+  it('should render service name with logout link', () => {
+    const { getByText } = render(
+      <UserContext.Provider
+        value={{
+          user: { name: 'bar' },
```

Comment:
Unless we need the name for something, I suggest we just leave this as an empty object

Reply:
I thought is better to have the name there as we're having it. But happy to leave it as an empty object